src/mcse.jl

```diff
@@ -1,9 +1,13 @@
-#################### Monte Carlo Standard Errors ####################
-
 """
-    mcse(x::AbstractVector{<:Real}; method::Symbol=:imse, kwargs...)
+    mcse(x::AbstractVector{<:Real}, method::Symbol=:imse; kwargs...)
+
+Return Monte Carlo Standard Errors of samples `x`.
```

Comment:
Maybe keep
```suggestion
Compute the Monte Carlo standard error (MCSE) of samples `x`.
```
since it was supposed to improve the existing docstring in MCMCChains.